kcidb/monitor/test_output.py

```diff
@@ -2,11 +2,10 @@
 
 import email
 import unittest
-from kcidb.io import schema
 from kcidb import orm, db, oo
 from kcidb.monitor.output import NotificationMessage, Notification,\
     NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN, NOTIFICATION_MESSAGE_BODY_MAX_LEN
-
+from kcidb.io import SCHEMA
```

Comment:
Ah, there's another line being removed unnecessarily, but let's get this merged already :joy: Otherwise it's perfect :smile:

Reply:
Ah, I managed to skip that 😃 

Thank you